DS-Session-1/dataSructure/TestQueue.java

```diff
@@ -0,0 +1,187 @@
+package dataSructure;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class TestQueue
+{
+	ArrayImplementationOfQueue queueUsingArray = new ArrayImplementationOfQueue(5);
+	LinkedListImplementationOfQueue queueUsingLinkedList = new LinkedListImplementationOfQueue();
+	
+	@Test
+	public void testPushArray()
+	{
+		queueUsingArray.push(3);
+		queueUsingArray.push(6);
+		assertEquals(3, queueUsingArray.peek());
+	}
+	
+	@Test
+	public void testPushArrayNegative()
+	{
+		queueUsingArray.push(3);
+		queueUsingArray.push(6);
+		assertNotEquals(6, queueUsingArray.peek());
+	}
+	
+	@Test
+	public void testPopArray()
+	{
+		queueUsingArray.push(3);
+		queueUsingArray.push(6);
+		assertEquals(3, queueUsingArray.pop());
+		assertEquals(6, queueUsingArray.peek());
+	}
+	
+	@Test
+	public void testPopArrayNegative()
+	{
+		queueUsingArray.push(3);
+		queueUsingArray.push(6);
+		assertNotEquals(6, queueUsingArray.pop());
+		assertEquals(6, queueUsingArray.peek());
+	}
+	
+	@Test
+	public void testPushArrayIfFull()
+	{
+		try
+		{
+			queueUsingArray.push(3);
+			queueUsingArray.push(6);
+			queueUsingArray.push(5);
+			queueUsingArray.push(7);
+			queueUsingArray.push(2);
+			queueUsingArray.push(9);
+		}	
+		catch(AssertionError e)
+		{
+			System.out.println(e);
```

Comment:
What is use of `System.out.println` in test methods? (Applicable to all the test methods like this)